ui-components/src/global/forms.scss

```diff
@@ -108,7 +107,7 @@
     display: inline-block;
     height: 1.25rem;
     line-height: 0.8;
-    margin-right: 0.8em;
+    margin-inline-end: 0.8em;
```

Comment:
I wonder if there's some linter that can catch these kinds of mistakes? Android layout linters will warn you about this kind of stuff.

Reply:
Probably outside the scope of this PR. :smile: